ERP-BACKEND/app/models/base.py

```
@@ -0,0 +1,31 @@
+from sqlalchemy.dialects.postgresql import JSONB
+from sqlalchemy.sql import func
+from app.database import Base
+
+
+# Use JSON for SQLite compatibility in tests, JSONB for PostgreSQL in production
+try:
+    # Test if we're using PostgreSQL
+    from sqlalchemy import create_engine
+    test_engine = create_engine("sqlite:///:memory:")
+    USE_JSONB = False
+except:
```

Ruff (0.16.1)

[error] Do not use bare `except`

(E722)